src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -187,7 +187,7 @@ export interface IAxisData {
 export interface IStringAxisData {
     data: {
         uniqueSampleKey: string;
-        value: string | (string[]);
```

Comment:
[Don't require parens for same-operator logical expressions](https://prettier.io/blog/2019/11/09/1.19.0.html#dont-require-parens-for-same-operator-logical-expressions-6864-by-jridgewell)